packages/node_modules/@webex/internal-plugin-devices/test/integration/spec/devices.js

```diff
@@ -1,21 +1,59 @@
 import '@webex/internal-plugin-devices';
 
 import {assert} from '@webex/test-helper-chai';
+import sinon from 'sinon';
 import testUsers from '@webex/test-helper-test-users';
+import WebexCore from '@webex/webex-core';
+
+import dto from './wdm-dto';
 
 describe('plugin-devices', () => {
   describe('Devices', () => {
+    let devices;
     let user;
+    let webex;
 
     before('create users', () => testUsers.create({count: 1})
       .then(([createdUser]) => {
         user = createdUser;
       }));
 
-    beforeEach('create the webex instance', () => {});
+    beforeEach('create the webex instance', () => {
+      webex = new WebexCore({
+        credentials: user.token
+      });
+
+      devices = webex.internal.devices;
+    });
+
+    // A later pull request will implement this method once this test utilizes
+    // `register()`.
     afterEach('attempt to unregister the device', () => {});
+
     after('remove users', () => testUsers.remove([user]));
 
+    describe('events', () => {
```

Comment:
Could these live in unit tests? They aren’t doing any network requests.

Reply:
moved.